core/__init__.py

```diff
@@ -1,5 +1,26 @@
 import os
 
+AGENT_DAEMON_NAMES = {
```

Comment:
why has this been moved here? this belongs in its original place in core.plugins.openstack since it is plugin specific code.

Reply:
Oh now i see why its cause you are implementing your code in core.checks. You should avoid putting plugin-specific code under core (outside of core.plugins). My advice would be to first implement your change in the openstack plugin itself then think about what elements of your implementation can be generalised and moved into core.* if needs be.